frontend/test/pytest/test_transform.py

```diff
@@ -46,7 +47,49 @@
 from catalyst import measure, qjit
 from catalyst.utils.exceptions import CompileError
 
-# pylint: disable=unnecessary-lambda-assignment
+# pylint: disable=too-many-lines,line-too-long
+
+
+@pytest.mark.xfail(reason="Noise models not supported on Lightning.")
+def test_add_noise(backend):
```

Comment:
Where `backend` is defined?

Reply:
The backend is a variable provided from the command line to pytest. It defaults to "lightning.qubit" but we also run it with "lightning.kokkos".